From d04aa4137d3e7d6301f2bf5c43511e312e653b88 Mon Sep 17 00:00:00 2001
From: Wu Liangqing <wlq@rock-chips.com>
Date: Fri, 29 May 2020 18:01:19 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3399-evb-ind-lpddr4-android-avb and
 rk3399-sapphire-excavator-edp add route_hdmi for hdmi boot logo

Change-Id: I8de8846b6c35247850c9480219bc94c2d0ec3c89
Signed-off-by: Wu Liangqing <wlq@rock-chips.com>
---
 .../boot/dts/rockchip/rk3399-evb-ind-lpddr4-android-avb.dts  | 5 +++++
 .../boot/dts/rockchip/rk3399-sapphire-excavator-edp.dtsi     | 5 +++++
 2 files changed, 10 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-ind-lpddr4-android-avb.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-ind-lpddr4-android-avb.dts
index e42b9cdeaa3c..e70d008626c1 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-ind-lpddr4-android-avb.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-ind-lpddr4-android-avb.dts
@@ -308,6 +308,11 @@
 	status = "okay";
 };
 
+&route_hdmi {
+	status = "okay";
+	connect = <&vopl_out_hdmi>;
+};
+
 &i2s1 {
 	#sound-dai-cells = <0>;
 	status = "okay";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dtsi
index 31c5a0ef4369..85957178dfb6 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dtsi
@@ -447,6 +447,11 @@
 	status = "okay";
 };
 
+&route_hdmi {
+	status = "okay";
+	connect = <&vopl_out_hdmi>;
+};
+
 &rt5651_sound {
 	status = "okay";
 };
-- 
2.35.3

